modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItDataTypesTest.java

```diff
@@ -322,7 +322,7 @@ public void testDecimalLiteral() {
                 "SELECT DECIMAL '0.09'  BETWEEN DECIMAL '0.06' AND DECIMAL '0.07'")
                 .returns(false).check();
 
-        assertQuery("SELECT ROUND(DECIMAL '10.000', 2)").returns(new BigDecimal("10.00")).check();
+        assertQuery("SELECT ROUND(DECIMAL '10.000', 2)").returns(new BigDecimal("10.000")).check();
```

Comment:
probably we need to add more accurate test:
assertQuery("SELECT TYPEOF(ROUND(DECIMAL '10.000', 2))").returns("DECIMAL(5, 3)").check();
wdyt ?

Reply:
honestly i forgot about our discussion for such a case, believe that type need to be the same in such a case too.

Reply:
This PR adds test cases for ROUND function to `ItFunctionsTest`. Test cases with return type assertions can be found there.